src/Entities/PatientData.php

```diff
@@ -0,0 +1,3007 @@
+<?php
+/**
```

Comment:
remember to only include the entities that are used in this PR (it seems like there are more than the last PR)

Reply:
removed

Reply:
I put it back because it is used in the PatientPharmacyController line 13